app/src/main/res/values/strings.xml

```diff
@@ -468,8 +468,8 @@ Upload your first media by tapping on the add button.</string>
   <string name="check_category_adding_template">Adding template for category check</string>
   <string name="check_category_notification_title">Requesting category check for %1$s</string>
     <string name="check_category_edit_summary">Requesting category check</string>
-  <string name="check_category_success_title">Requesting category check: Success</string>
-  <string name="check_category_failure_title">Requesting category check: Failed</string>
+  <string name="check_category_success_title">Category check requested</string>
```

Comment:
How about "Category check successfully requested"?

Reply:
Ok, but what about fail message? Category check request didn't work.

Reply:
The next line that you altered (`"check_category_failure_title"`) is fine :)